tools/plugins/com.liferay.ide.gradle.core/src/com/liferay/ide/gradle/core/ModuleFragmentProjectProvider.java

```diff
@@ -230,6 +238,7 @@ public IStatus createNewProject( NewModuleFragmentOp op, IProgressMonitor monito
             else
             {
                 GradleUtil.importGradleProject( projecLocation.toFile(), monitor );
+                addLinkForTemplate(projectName,temp);
```

Comment:
Lets not create the link automatically when project is imported.  Instead lets create the link just before we need to open the compare editor.  And then remove the link after the editor is closed.